.ci/deps.sh

```diff
@@ -95,8 +95,10 @@ if [[ -z "$(which hlint)" ]]; then
 fi
 
 # cabal update to 1.22.9.0 and install ghc-mod 5.6.0
-cabal update && cabal install cabal-install-1.22.9.0
-cabal install ghc-mod
+if [[ -z "$(which ghc-mod)" ]]; then
```

Comment:
@jayvdb but this is not checking if it complies to the version provided in the GhcModBear?!
`cabal-instal-1.22.9.0` will install `ghc-mod-5.6.0`